### ISSUE TITLE
shorten ninja spawn back to 20minutes

### ISSUE THREAD

**Issue by ToGWtF** [yogstation13/yogstation#1685]
FUCKING SPAWNED AS NINJA WITH 3 MINUTES TO SHUTTLE DOCK RREEEEEEEEEEEE

**Comment by ShadowDeath6** [comment]
Ninja needs a spawn weight lower than zero.

**Comment by Dagdammit** [comment]
I recently got a chance to play as space ninja for the first time ever. It was about 60 seconds before the escape shuttle launched.

I was actually floored to discover how much work had gone into it, the flavor was off the charts. Hell all that biometric data alone was amazing.

**Comment by Lightning38088** [comment]
yeah we rarely see ninjas anymore and if we do its as the shuttle launches

**Comment by Dagdammit** [comment]
Literally saw one this morning and it was as the shuttle was coming.
